### PR TITLE
build: Remove packageVersion files

### PR DESCRIPTION
**Pull request by tylerbutler**
There are a lot of packages that generate a packageVersion.ts file but that file is not used. I removed all the unused files and updated the build scripts acordingly.

### CODE REVIEW

File: build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts [hunk 1]
```diff
@@ -293,7 +293,7 @@ export class FluidPackageCheck {
 			const buildPrefix = pkg.getScript("build:gen")
 				? "npm run build:gen && "
 				: pkg.getScript("build:genver")
-				? "npm run build:genver && "
+				? ""
```
Comment (tylerbutler) [inline build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts:296]:
```suggestion
				? "npm run build:genver && "
```

File: package.json [hunk 2]
```diff
@@ -12,19 +12,19 @@
 	"author": "Microsoft and contributors",
 	"scripts": {
 		"build": "npm run policy-check && npm run layer-check && npm run build:genver && npm run build:compile && npm run lint && npm run build:docs",
-		"build:ci": "npm run build:genver && lerna run build:compile --stream",
+		"build:ci": "lerna run build:compile --stream",
```
Comment (alexvy86) [inline package.json:15]: It shouldn't stay here so that build the whole repo still generates the version file for the packages that care about it?

**Review by alexvy86 — APPROVED** [review]
Overall I don't see anything obviously wrong (except the one comment below, which might also apply to the other release group roots)... but I have the question of what is that version file actually used for, then? How do we know which packages need it or don't?

File: package.json (hunk 2 above)
Comment (tylerbutler) [reply package.json]: You're absolutely right! I reverted all the release group root changes.

**Comment by tylerbutler** [comment]
> I have the question of what is that version file actually used for, then? How do we know which packages need it or don't?

I considered them "used" if the build broke when they were removed. I added them back one at a time until the build passed. My hunch is that a project was copied at some point and the files were assumed to be needed/used, so they were left.

I included a list of all the remaining files so you can see where they're used. DDSes, drivers, and the runtime all stick out as common categories. I can sort of understand how those things would benefit from having a version available in code. But given our internal version scheme, the only reasonable thing that code can do is compare them and see if one is higher/lower.

```
azure/packages/test/scenario-runner/src/packageVersion.ts
experimental/dds/attributable-map/src/packageVersion.ts
experimental/dds/ot/ot/src/packageVersion.ts
experimental/dds/ot/sharejs/json1/src/packageVersion.ts
experimental/dds/sequence-deprecated/src/packageVersion.ts
packages/dds/cell/src/packageVersion.ts
packages/dds/counter/src/packageVersion.ts
packages/dds/ink/src/packageVersion.ts
packages/dds/map/src/packageVersion.ts
packages/dds/matrix/src/packageVersion.ts
packages/dds/ordered-collection/src/packageVersion.ts
packages/dds/pact-map/src/packageVersion.ts
packages/dds/register-collection/src/packageVersion.ts
packages/dds/sequence/src/packageVersion.ts
packages/dds/shared-object-base/src/packageVersion.ts
packages/dds/shared-summary-block/src/packageVersion.ts
packages/dds/task-manager/src/packageVersion.ts
packages/drivers/driver-base/src/packageVersion.ts
packages/drivers/odsp-driver/src/packageVersion.ts
packages/drivers/routerlicious-driver/src/packageVersion.ts
packages/loader/container-loader/src/packageVersion.ts
packages/loader/driver-utils/src/packageVersion.ts
packages/runtime/container-runtime/src/packageVersion.ts
packages/test/mocha-test-setup/src/packageVersion.ts
packages/test/snapshots/src/packageVersion.ts
packages/test/test-drivers/src/packageVersion.ts
packages/test/test-end-to-end-tests/src/packageVersion.ts
packages/test/test-service-load/src/packageVersion.ts
packages/test/test-utils/src/packageVersion.ts
packages/test/test-version-utils/src/packageVersion.ts
packages/utils/odsp-doclib-utils/src/packageVersion.ts
packages/utils/tool-utils/src/packageVersion.ts
server/routerlicious/packages/memory-orderer/src/packageVersion.ts
server/routerlicious/packages/services/src/packageVersion.ts
server/routerlicious/packages/services-client/src/packageVersion.ts
server/routerlicious/packages/services-core/src/packageVersion.ts
server/routerlicious/packages/services-ordering-kafkanode/src/packageVersion.ts
server/routerlicious/packages/services-shared/src/packageVersion.ts
server/routerlicious/packages/services-utils/src/packageVersion.ts
```

**Pull request merged**